src/JitInterface/src/JitConfigProvider.cs

```diff
@@ -105,14 +105,14 @@ private unsafe IntPtr CreateUnmanagedInstance()
             return instance;
         }
 
-        [UnmanagedFunctionPointer(CallingConvention.ThisCall)]
+        [UnmanagedFunctionPointer((CallingConvention)0)]
```

Comment:
Nitpicking: I would find `default(CallingConvention)` prettier (if the intent here is to give the 0 a semantic - or just a 0 without a cast otherwise, because the cast is not needed).